Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,35 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
```

Comment:
I think we know it's open source …

Reply:
Agreed and fixed. (It’s almost as if you were slowly running out of nits …)